### DISCUSSION TITLE
Multiple Connection panels

### DISCUSSION THREAD

**Discussion by Klauss-Preising** [isontheline/pro.webssh.net#421] [Ideas]
It would be cool if we could have like 2 or 3 panels where we can see different connections, or create different windows for connections. maybe we could ssh to one and sftp in another to the same host.

**Comment by isontheline** [comment]
Could you check this?
https://webssh.net/documentation/help/howtos/launching-multiple-terminals/

**Reply by Klauss-Preising** [reply]
I meant so we can see the connections side by side if that makes sense.

**Reply by isontheline** [reply]
OK makes sense 😇

An issue is already open for Split View Support : https://github.com/isontheline/pro.webssh.net/issues/151
